src/main/kotlin/TaskFactory.kt

```diff
@@ -76,6 +83,29 @@ private fun Project.registerDockerBuildTask(
     return dockerBuildTask
 }
 
+private fun Project.registerDockerScanTask(
+    profile: ValidatedDockerScanProfile,
+    dockerBuildTask: String,
+    serviceName: String,
+    taskSuffix: String
+): String {
+    val dockerScanTask = "dockerScan$taskSuffix${serviceName.capitalize()}"
+    tasks.register(dockerScanTask, DockerGrypeScanTask::class.java) {
+        it.failOnThreshold = profile.failOnThreshold
+        if (profile.dockerDir != null) {
+            it.buildDockerDir = profile.dockerDir
+        }
+        it.ignoreFilePath = profile.ignoreFilePath
+        it.serviceName = serviceName
+        it.dependsOn(dockerBuildTask)
+    }
+
+    tasks.named("scan") {
```

Comment:
This task is not present per default. Options: Omit this task or register it separately